### PR TITLE
Jose summer test2

### PR DESCRIPTION
**Pull request by Josee1031**

### CODE REVIEW

**Comment by markcmiller86** [comment]
@Josee1031 it looks like you may have inadvertently included several files that should not be commited. All the `.o` files for example. Maybe a stray `git commit -a` was the cause (`-a` means all) or a `git add *`). Also, you included a results file from a run  `heat_results` drectory. Please remove those from the commit.

**Comment by markcmiller86** [comment]
@Josee1031 can you please cut-n-paste the actual warning text you were getting here?

**Comment by markcmiller86** [comment]
pinging @remontel here so he can follow along as needed.

**Comment by Josee1031** [comment]
> @Josee1031 can you please cut-n-paste the actual warning text you were getting here?

./Number.H:30:9: warning: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Wdeprecated-declarations]
        sprintf(counts, "Adds:%d, Mults:%d, Divs:%d, Bytes:%d", Number::nadds, Number::nmults, Number::ndivs, (int) Number::nbytes);

**Comment by markcmiller86** [comment]
Hmmm...that looks like a *new* warning with your changes from `snprintf` to `sprintf`. It was using `snprintf` before. Can you revert your changes and produce the error message the code was originally emitting?

**Comment by Josee1031** [comment]
I was referring to the original bug. I fixed it on the original PR by changing it to snprintf and establishing a buffer size. The issue was that I pushed without using "make clean" and deleting the heat_results folder from the tutorial, that's why there were .o files and a heat_results folder. I made sure to remove them on the second PR.

File: Number.H [hunk 1]
```diff
@@ -27,7 +27,7 @@ class Number {
     static void *operator new(std::size_t sz) { Number::nbytes += sz; return ::operator new(sz); };
     static void *operator new[](std::size_t sz) { Number::nbytes += sz; return ::operator new(sz); };
     static char const *counts_string() { static char counts[256];
-        sprintf(counts, "Adds:%d, Mults:%d, Divs:%d, Bytes:%d", Number::nadds, Number::nmults, Number::ndivs, (int) Number::nbytes);
+        snprintf(counts,sizeof(counts), "Adds:%d, Mults:%d, Divs:%d, Bytes:%d", Number::nadds, Number::nmults, Number::ndivs, (int) Number::nbytes);
```
Comment (markcmiller86) [inline Number.H]:
```suggestion
        snprintf(counts, sizeof(counts), "Adds:%d, Mults:%d, Divs:%d, Bytes:%d", Number::nadds, Number::nmults, Number::ndivs, (int) Number::nbytes);
```

File: args.C [hunk 2]
```diff
@@ -161,7 +161,7 @@ process_args(int argc, char **argv)
     // Make the output dir and save clargs there too
     mkdir(runame, S_IRWXU|S_IRWXG|S_IROTH|S_IXOTH);
     char fname[128];
-    sprintf(fname, "%s/clargs.out", runame);
+    snprintf(fname,sizeof(fname), "%s/clargs.out", runame);
```
Comment (markcmiller86) [inline args.C]:
```suggestion
    snprintf(fname, sizeof(fname), "%s/clargs.out", runame);
```

**Review by markcmiller86 — CHANGES_REQUESTED** [review]
Ok, much better. Thanks for fixing this. I proposed a minor change to each line to keep space after comma styling. You can accept those by hitting the approprate button under the proposed change.

**Review by markcmiller86 — APPROVED** [review]
Thank you

**Pull request merged**